tests/new_unit_tests/Commands/Connection/ConnectionCommandTest.php

```diff
@@ -0,0 +1,41 @@
+<?php
+/**
+ * @file
+ *
+ * Contains Pantheon\Terminus\UnitTests\Commands\Connection\ConnectionCommandTest
+ */
+
+namespace Pantheon\Terminus\UnitTests\Commands\Connection;
+
+
+use ReflectionMethod;
+
+abstract class ConnectionCommandTest extends \PHPUnit_Framework_TestCase
```

Comment:
testing only the public interface is a common strategy. I am still a bit stumped on how we want to test commands via unit tests. Commands should be super high level and devoid of much logic by design. They should mostly just pass the input off to a service that returns output ready to be provided to the user. They should sort of be like a controller, a thin layer of glue code.